src/components/abstractComponent.ts

```diff
@@ -226,7 +223,7 @@ export module Component {
      * @param {number} [availableHeight] - the height of the container element
      * @returns {Component} The calling component.
      */
-    public resize(width?: number, height?: number): AbstractComponent {
+    public forceRedraw(width?: number, height?: number): AbstractComponent {
       if (!this._isTopLevelComponent) {
         throw new Error("Cannot resize on non top-level component");
       }
```

Comment:
Attempting to force a redraw on a nested component might not make sense? That's my best guess for why this was here before.